plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/bls12_381_G1_add/add-zero/add-zero.uplc

```diff
@@ -0,0 +1,10 @@
+-- Adding the zero element to a random point doesn't change it.
+(program 1.0.0
+ [
+  [
+   (builtin bls12_381_G1_add)
+   (con bls12_381_G1_element 0xabd61864f519748032551e42e0ac417fd828f079454e3e3c9891c5c29ed7f10bdecc046854e3931cb7002779bd76d71f)
+  ]
+  (con bls12_381_G1_element 0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000)
```

Comment:
I wondered about that, but maybe it's better to have the program output something that you can actually look at rather than just True/False.  Not sure.

Reply:
> I wondered about that, but maybe it's better to have the program output something that you can actually look at rather than just True/False. Not sure.

Yeah, good point.